packages/x-charts/package.json

```diff
@@ -40,8 +40,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.25.0",
-    "@mui/system": "^5.16.5",
-    "@mui/utils": "^5.16.5",
+    "@mui/system": "^6.0.0-beta.5",
+    "@mui/utils": "^6.0.0-beta.5",
```

Comment:
This might be problematic.
Is it OK for us to use v6 here if users are going to use it with `@mui/material@5`?

cc @LukasTy @DiegoAndai

Reply:
I was experimenting with it.
AFAIK, reverting these versions to v5 and adding a `devDependency` on v6 resulted in v5 still being used and some new test failures.
Maybe after all the recent changes it would be possible to use v5 for these deps. 🤞